src/ensembl/src/content/app/browser/track-panel/track-panel-modal/TrackPanelModal.scss

```diff
@@ -39,18 +39,34 @@
 }
 
 .previouslyViewed {
-  margin-top: 60px;
+  margin-top: 30px;
 
   a {
     font-weight: $normal;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    max-width: 215px;
+    white-space: nowrap;
+    display: inline-block;
+    font-size: 13px;
   }
 
   dt {
     color: $ens-dark-grey;
     font-size: 12px;
+    border-bottom: 1px solid $ens-grey;
   }
 
   dd {
     margin: 15px 0 0 15px;
+    position: relative;
   }
 }
+
+.previouslyViewedType {
```

Comment:
What does "previouslyViewedType" mean, and how is it different from "previouslyViewed" above?

Reply:
"previouslyViewedType" is the greyed out type display next to the link and "previouslyViewed" is the actual text used for the link.